applications/smart-distancing/docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I am not familiar with Sphinx, nor with Windows :D 
But it seems targeted for Windows OS which just does not make sense to be part of the whole project as it won't (can't) work on that platform. 
Just suggesting not pushing unnecessary things as the PR itself is pretty big.